test/lib/ufe/testDeleteCmd.py

```diff
@@ -138,3 +140,100 @@ def childrenNames(children):
 
         self.assertNotIn(sphereShapeName, sphereChildrenNames)
         self.assertNotIn(ball35Name, propsChildrenNames)
+
+        # undo to restore state to original.
+        cmds.undo()
+
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2011', 'testVisibility only available in UFE preview 2011 and greater')
+    def testDeleteArgs(self):
+        '''Delete Maya and USD objects passed as command arguments.'''
```

Comment:
Pretty much the same test as delete using selection, only in this case the UFE arguments are passed as UFE path strings.